evm/src/main/java/com/horizen/evm/interop/EvmResult.java

```diff
@@ -3,25 +3,20 @@
 import com.horizen.evm.utils.Address;
 
 import java.math.BigInteger;
-import java.util.Map;
 
 public class EvmResult {
     public BigInteger usedGas;
     public String evmError;
     public byte[] returnData;
     public Address contractAddress;
-    public EvmTraceLog[] traceLogs;
-    public Map<String,Integer> fourByteTracerLogs;
-    public EvmCallTraceLog callTracerLogs;
     public Boolean reverted;
+    public Object tracerResult;
 
     public static EvmResult emptyEvmResult() {
         var evmResult = new EvmResult();
-
         evmResult.usedGas = BigInteger.ZERO;
         evmResult.returnData = new byte[] {0};
```

Comment:
Unrelated to your changes: Why do we have have `new byte[] {0}` here? I assume it should be an empty array, i.e. `new byte[0]`.

Reply:
This constructor was introduced in another PR to resolve some `nullPointerException` in case we call the rpc method for transactions that doesn't involve the evm.
It was an hotfix, I remove it